MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Clock.xaml

```diff
@@ -547,4 +759,2919 @@
+                                        </StackPanel.Resources>
+
+                                        <Grid Width="96" Height="80" x:Name="PART_HourReadOut">
+                                            <Border x:Name="PART_HourReadOutBorder"
+                                                    Opacity=".12"
+                                                    CornerRadius="{TemplateBinding CornerRadius}">
+                                                <Border.Style>
+                                                    <Style TargetType="Border">
+                                                        <Setter Property="Background" Value="{DynamicResource MaterialDesignBodyLight}"/>
+                                                        <Style.Triggers>
+                                                            <DataTrigger Binding="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=DisplayMode}" Value="{x:Static wpf:ClockDisplayMode.Hours}">
+                                                                <Setter Property="Background" Value="{DynamicResource PrimaryHueMidBrush}" />
+                                                            </DataTrigger>
+                                                        </Style.Triggers>
+                                                    </Style>
+                                                </Border.Style>
+                                            </Border>
+                                            <TextBlock Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Time, Mode=OneWay, StringFormat={}{0:%h}}" 
+													   x:Name="PART_HourReadOutText"
+                                                       HorizontalAlignment="Center"
+                                                       VerticalAlignment="Center">
+                                                <TextBlock.Style>
+                                                    <Style BasedOn="{StaticResource TimeTextBlock}" TargetType="{x:Type TextBlock}">
+                                                        <Style.Triggers>
+                                                            <DataTrigger Binding="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=DisplayMode}" Value="{x:Static wpf:ClockDisplayMode.Hours}">
+                                                                <Setter Property="Foreground" Value="{DynamicResource PrimaryHueMidBrush}" />
+                                                            </DataTrigger>
+                                                        </Style.Triggers>
+                                                    </Style>
+                                                </TextBlock.Style>
+                                            </TextBlock>
+                                        </Grid>
+
+                                        <TextBlock Text=":" Style="{StaticResource TimeTextBlock}"
```

Comment:
Same comment here. Would be good to use a string format specifier to get a localized string.